src/pynwb/ndx_microscopy/testing/_mock.py

```diff
@@ -7,73 +7,32 @@
 import ndx_microscopy
 
 
-def mock_Microscope(
+def mock_Microscopy(
     *,
     name: Optional[str] = None,
-    description: str = "This is a mock instance of a Microscope type to be used for rapid testing.",
-    manufacturer: str = "A fake manufacturer of the mock microscope.",
-    model: str = "A fake model of the mock microscope.",
-) -> ndx_microscopy.Microscope:
-    microscope = ndx_microscopy.Microscope(
-        name=name or name_generator("Microscope"),
+    description: str = "This is a mock instance of a Microscopy type to be used for rapid testing.",
+) -> ndx_microscopy.Microscopy:
+    microscopy = ndx_microscopy.Microscopy(
+        name=name or name_generator("Microscopy"),
         description=description,
-        manufacturer=manufacturer,
-        model=model,
     )
-    return microscope
+    return microscopy
```

Comment:
Did you mean to do this?

Reply:
No sorry, I forgot to put a TODO comment, it was a work in progress